report/report.go

```diff
@@ -289,6 +290,17 @@ func PrintObjectCounts(reportFile io.WriteCloser, objectCounts map[string]int) {
  * users will never use a +dev version in production.
  */
 func EnsureBackupVersionCompatibility(backupVersion string, restoreVersion string) {
+	if strings.Index(restoreVersion, "_arenadata") != -1 {
```

Comment:
Do we consider a case when the restore version may not consist `_arenadat` suffix? Maybe replace it with assertion check?

Reply:
If the restore version doesn't contain _arenadat suffix, we will fail during the original flow of pivotal's checks. (By this I suppose this solution only works when we restore with arenadata build) Is it too ugly?

Reply:
As we don't have tests to restore "arenadata" backup with upstream gprestore (without suffix) and don't plan to maintain such configurations, this comment doesn't matter. Moreover, if we fail on restore without suffix, we'll break down tests for branches with upstream changes. These branches are containing the newest tag from the upstream without a suffix.